src/framework/components/anim/component-layer.js

```diff
@@ -199,6 +208,19 @@ class AnimComponentLayer {
         return this._mask;
     }
 
+    /**
+     * Whether this layers weight should be normalized by the total weight of all layers or calculated separately.
+     *
+     * @type {boolean}
+     */
+    set normalizedWeight(value) {
+        this._normalizedWeight = value;
```

Comment:
looking at `set blendType`, it feels like maybe `rebind()` should be called from here too in some case?

Reply:
Sorry this was an old function. It's been moved to the component class and it resets the state graph after being set.